client/jetpack-connect/plans-extended-info.jsx

```diff
@@ -0,0 +1,55 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormattedHeader from 'components/formatted-header';
+
+const PlansExtendedInfo = ( { onClick, translate } ) => {
+	const headerText = translate( 'Unsure which plan is right for you?' );
+	const subheaderText = translate(
+		'Read out guide to choosing the right plan or compare plans in more detail.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Read our guide to choosing the right plan or compare plans in more detail.' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
There's a typo here, `out` vs `our`.

Reply:
ooops! Thanks!